readers/influxdb/messages.go

```diff
@@ -90,11 +117,8 @@ func genMessage(names []string, fields []interface{}) mainflux.Message {
 			u, _ := strconv.ParseUint(fields[i].(string), 10, 64)
 			msgField.SetUint(u)
 		case float64:
-			msgField.SetFloat(parseFloat(fields[i]))
-		case bool:
-			if b, ok := fields[i].(bool); ok {
-				msgField.SetBool(b)
-			}
+			val, _ := strconv.ParseFloat(fields[i].(string), 64)
```

Comment:
Please handle this error.

Reply:
Based on the private discussion we've had, I'll mark this as resolved.
> This obscure way of handling InfluxDB read is caused by strange API of the InfluxDB official Go client and should be updated as soon as the official client gets support for mapping data to custom struct. Also, this query could be optimized, but it can be a part of the aforementioned PR sometime in the future.